views/index.hbs

```diff
@@ -1,3 +1,5 @@
 <div>
-
+    <img src="/images/beer.png" alt="beer-img">
+    <a href="/beers.hbs">Check the Beers!</a>
```

Comment:
probably this link dont work rigth ? , you´ll need to remove the .hbs on the href.